packages/base/src/asset-registries/Icons.ts

```diff
@@ -138,6 +139,37 @@ const getIconData = async (name: string) => {
 	return registry.get(registryKey);
 };
 
+/**
+ * Returns the accessible name for the given icon,
+ * or undefined if accessible name is not present.
+ *
+ * @param { string } name
+ * @return { Promise }
+ */
+const getIconAccessibleName = async (name: string) => {
+	let iconData: typeof ICON_NOT_FOUND | IconData | undefined = getIconDataSync(name);
+	if (!iconData) {
+		iconData = await getIconData(name);
+	}
+
+	if (!iconData) {
+		/* eslint-disable-next-line */
+		console.warn(`Required icon is not registered. Invalid icon name: ${name}`);
+		return;
+	}
+
+	if (iconData === ICON_NOT_FOUND) {
+		/* eslint-disable-next-line */
+		console.warn(`Required icon is not registered. You can either import the icon as a module in order to use it e.g. "@ui5/webcomponents-icons/dist/${name.replace("sap-icon://", "")}.js", or setup a JSON build step and import "@ui5/webcomponents-icons/dist/AllIcons.js".`);
+		return;
+	}
+
+	if (iconData && iconData.accData) {
+		const i18nBundle = await getI18nBundle(iconData.packageName);
+		return i18nBundle.getText(iconData.accData) || undefined;
```

Comment:
not sure why the last `|| undefined` is necessary

Reply:
We can remove it, there is a case that `getText` returns empty string and I think we should not add acc attrs ( that expects value) to an empty string, they should be rather not set. But, this can be handled outside.